draft-ietf-jsonpath-base.md

```diff
@@ -1234,30 +1222,39 @@ The `filter-selector` works with arrays and objects exclusively. Its result is a
 A relative path, beginning with `@`, refers to the current array element or member value as the
 filter selector iterates over the array or object.
 
-When a path resulting in an empty nodelist appears on either side of a comparison, the result of the comparison is
+A singular path by itself in a Boolean context is an existence test which yields true if the path selects a node and yields false if the path does not select a node.
```

Comment:
Did we ever clarify that we don't want existence tests on general paths?

Reply:
I was under the impression that the consensus was to restrict existence tests to singular paths. That seems to be clear in the spec. Did you want further clarification in the spec or to revisit the question?